dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,1097 @@
+    public static Long waitTimeBeforeStreamingJobStatusInTask = TestConfig.tab().longAt(SnappyPrms.streamingJobExecutionTimeInMillisForTask, 600000);
+    public static Long waitTimeBeforeJobStatusInCloseTask = TestConfig.tab().longAt(SnappyPrms.jobExecutionTimeInMillisForCloseTask, 600000);
+    public static final int THOUSAND = 1000;
+    public static String cycleVMTarget = TestConfig.tab().stringAt(SnappyPrms.cycleVMTarget, "store");
+    private static Set active;
+
+    public static <A, B> Map<A, B> toScalaMap(HashMap<A, B> m) {
+        return JavaConverters.mapAsScalaMapConverter(m).asScala().toMap(Predef.<Tuple2<A, B>>conforms());
+    }
+
+    public static void HydraTask_initializeSnappy() {
+/*        org.apache.spark.SparkConf conf = new org.apache.spark.SparkConf().setAppName("SnappyTest").setMaster("local").set("spark.ui.port", "6060");
+        jsc = new JavaSparkContext(conf);
+        snc = SnappyContext.getOrCreate(JavaSparkContext.toSparkContext(jsc));*/
+        SparkContext sc = null;
+        snc = SnappyContext.getOrCreate(sc);
+        Log.getLogWriter().info("SS - SnappyContext initialized successfully");
+    }
+
+
+    public static void HydraTask_stopSnappy() {
+        snc.stop();
+        Log.getLogWriter().info("SS - SnappyContext stopped successfully");
+
+    }
+
+    public static void HydraTask_createRowColumnTableUsingSQL() {
+        HydraTask_initializeSnappyTest();
+        String hfile = snappyTest.getDataLocation() + "/airlineParquetData";
+        String reffile = snappyTest.getDataLocation() + "/airportcodeParquetData";
+
+        snc.sql("CREATE TABLE " + stagingColumnTable + " \n" +
+                " USING parquet OPTIONS(path '" + hfile + "')");
+        Log.getLogWriter().info("SS - staging column table " + stagingColumnTable + " created successfully");
+        snc.sql("CREATE TABLE " + columnTable + " USING column OPTIONS(buckets '11') AS (\n" +
+                "  SELECT Year AS yeari, Month AS monthi , DayOfMonth,\n" +
+                "    DayOfWeek, DepTime, CRSDepTime, ArrTime, CRSArrTime,\n" +
+                "    UniqueCarrier, FlightNum, TailNum, ActualElapsedTime,\n" +
+                "    CRSElapsedTime, AirTime, ArrDelay, DepDelay, Origin,\n" +
+                "    Dest, Distance, TaxiIn, TaxiOut, Cancelled, CancellationCode,\n" +
+                "    Diverted, CarrierDelay, WeatherDelay, NASDelay, SecurityDelay,\n" +
+                "    LateAircraftDelay, ArrDelaySlot\n" +
+                "    FROM " + stagingColumnTable + ")");
+        Log.getLogWriter().info("SS - column table " + columnTable + " created successfully");
+        snc.sql("CREATE TABLE " + stagingRowTable + " \n" +
+                "  USING parquet OPTIONS(path '" + reffile + "')");
+        Log.getLogWriter().info("SS - staging row table " + stagingRowTable + " created successfully");
```

Comment:
even while creating the table with hardcoded data atleast option field can be taken by the user to facilitate different types of table creation as per user requirement